Before this PR, I would get below error when running parallel builds

```
make[2]: Entering directory `/home/devel/optee/repo/hello_world/ta'
make[2]: warning: jobserver unavailable: using -j1. Add `+' to parent make rule.
make[2]: *** No rule to make target `user_ta_header.c', needed by `user_ta_header.o'. Stop.
```

Is this a different error?

--- a/Makefile
+++ b/Makefile
@@ -3,10 +3,10 @@ export V?=0
 
 .PHONY: all
 all:
-	make -C host CROSS_COMPILE="$(HOST_CROSS_COMPILE)"
-	make -C ta CROSS_COMPILE="$(TA_CROSS_COMPILE)"
+	$(MAKE) -C host CROSS_COMPILE="$(HOST_CROSS_COMPILE)"
+	$(MAKE) -C ta CROSS_COMPILE="$(TA_CROSS_COMPILE)"
 
 .PHONY: clean
 clean:
-	make -C host clean
-	make -C ta clean
+	$(MAKE) -C host clean
+	$(MAKE) -C ta clean

